in_text/text_04_predictions.yml

```diff
@@ -76,794 +76,1075 @@ cross-cites:
-      attr-def: >-
-        Computed daily mean stream temperature for each segment
-      attr-defs: This data release
-      data-min: 0
-      data-max: 26.09
-      data-units: degrees Celsius
-    -
-      attr-label: seg_upstream_inflow
-      attr-def: >-
-        Sum of inflow from upstream segments
-      attr-defs: This data release
-      data-min: 0
-      data-max: 841.5766
-      data-units: cubic meters per second
-    -
-      attr-label: seg_width
-      attr-def: >-
-        Width of each segment.
-      attr-defs: This data release.
-      data-min: 1.622
-      data-max: 93.45
-      data-units: meters
-    -
-      attr-label: seginc_gwflow
-      attr-def: >-
-          Area-weighted average groundwater discharge for each segment from HRUs contributing flow to the segment.
-      attr-defs: This data release
-      data-min: 0
-      data-max: 12.46791
-      data-units: cubic meters per second
-    -
-      attr-label: seginc_potet
-      attr-def: >-
-        Area-weighted average potential ET for each segment from HRUs contributing flow to the segment.
-      attr-defs: This data release
-      data-min: 4.146e-08
-      data-max: 0.3052
-      data-units: meters
-    -
-      attr-label: seginc_sroff
-      attr-def: >-
-        Area-weighted average surface runoff for each segment from HRUs contributing flow to the segment.
-      attr-defs: This data release
-      data-min: 0
-      data-max: 88.00875
-      data-units: cubic meters per second
-    -
-      attr-label: seginc_ssflow
-      attr-def: >-
-        Area-weighted average interflow for each segment from HRUs contributing flow to the segment.
-      attr-defs: This data release
-      data-min: 0
-      data-max: 147.5874
-      data-units: cubic meters per second
-    -
-      attr-label: seginc_swrad
-      attr-def: >-
-        Area-weighted average solar radiation for each segment from HRUs contributing flow to the segment.
-      attr-defs: This data release
-      data-min: 0.0009744971
-      data-max: 341.4374
-      data-units: watts per square meter
-    -
-      attr-label: seg_length
-      attr-def: >-
-        Length of stream segment.
-      attr-defs: https://www.sciencebase.gov/catalog/item/535eda80e4b08e65d60fc834
-      data-min: 108.8776
-      data-max: 31063.61
-      data-units: meters
-    -
-      attr-label: seg_slope
-      attr-def: >-
-        Stream segment slope.
-      attr-defs: https://www.sciencebase.gov/catalog/item/535eda80e4b08e65d60fc834
-      data-min: 2e-06
-      data-max: 0.04883
-      data-units: NA
-    -
-      attr-label: seg_elev
-      attr-def: >-
-        Stream segment average elevation above mean sea level.
-      attr-defs: https://www.sciencebase.gov/catalog/item/535eda80e4b08e65d60fc834
-      data-min: 256.44
-      data-max: 618.24
-      data-units: meters
+      data-units: integer
   -
-    data-name: reservoir_interpolated_daily_water_budget_components.csv
+    data-name: all_mods_with_obs.rds
```

Comment:
the attributes and min max values entered here are from last year's `all_mods_with obs`. Must update once `all_mods_with_obs.rds` is found.